Frameworks/UIKit/StarboardXaml/StarboardXaml.h

```diff
@@ -58,7 +58,7 @@ ref class AppEventListener
 
 extern "C" void _ApplicationLaunch(ActivationType activationType, Platform::Object^ activationArg);
 extern "C" void EbrApplicationActivated(Windows::ApplicationModel::Activation::IActivatedEventArgs^ args);
```

Comment:
yes this will be a breaking change as this went in this week's release and apps imported with this week's vsimporter will break

---
In reply to: [72348258](https://github.com/Microsoft/WinObjC/pull/686#discussion_r72348258) [](ancestors = 72348258,72338427,72336614)